lang/cs/Org.Apache.REEF.Wake/Impl/TimerStage.cs

```diff
@@ -15,14 +15,19 @@
 // specific language governing permissions and limitations
 // under the License.
 
-using System.Timers;
+using System;
+using System.Threading;
 
 namespace Org.Apache.REEF.Wake.Impl
 {
     /// <summary>Stage that triggers an event handler periodically</summary>
     public sealed class TimerStage : IStage
     {
-        // private readonly ScheduledExecutorService executor;
+        enum TimeValue : long
```

Comment:
If we add a constant it should be public so users of the class can use the value too. This would also allow us to make the constant compile time dependent in the future since callers should now check against TimerStage.MaxTimeValue.